PixaPicker/URLExtensions.swift

```diff
@@ -10,7 +10,7 @@ import Foundation
 
 class URLExtensions {
     
-    static func pixabaySearchURL(withtext: String) -> String{
+    static func pixabaySearchURL(withtext: String, withPageNumber: Int) -> String{
```

Comment:
I'd write this as `with pageNumber: Int` (adding a space which you're allowed to do in Swift).  I guess that means also . `with text: String`